grails-app/domain/org/pih/warehouse/requisition/Requisition.groovy

```diff
@@ -117,16 +119,42 @@ class Requisition implements Comparable<Requisition>, Serializable {
 
     Integer statusSortOrder
 
+    // Request approval fields
+    Person approvedBy
+
+    Date dateApproved
+
+    Date dateRejected
```

Comment:
I don't mind doing that, so I leave the decision for you.
On the other hand, I think this could be misleading, unless we'd override the `dateSubmitted` over and over again.
I imagine a situation, when the request is REJECTED, then I believe someone will have a chance to correct it (?), and then again will send it for an approval (submit).

So the question would be - if we reject the request - should the `dateSubmitted` be cleared out or should it stay, and then, after another submission we'd just override it?

Reply:
Discussed with the guys - we have `dateRequested` already, hence we didn't find an advantage of adding it for now.